tests/testsuite/cargo_report_sessions/mod.rs

```diff
@@ -91,13 +82,10 @@ fn in_workspace() {
 
     p.cargo("report sessions -Zbuild-analysis")
         .masquerade_as_nightly_cargo(&["build-analysis"])
-        .with_status(1)
         .with_stderr_data(str![[r#"
-[ERROR] unrecognized subcommand 'sessions'
-
-Usage: cargo report [OPTIONS] <COMMAND>
+Session IDs for `[ROOT]/foo` (showing up to 10):
 
-For more information, try '--help'.
+ - [..]T[..]Z-[..]
```

Comment:
Should the output be solely for humans or also work for basic scripting?  **We can leave this for the tracking issue.**
- We could move the workspace and limit filters to a `note`